### ISSUE TITLE
[OS X 10.11.6] Can't compile master branch

### ISSUE THREAD

**Issue by lukaville** [libimobiledevice/ideviceinstaller#55]
```
➜  ideviceinstaller git:(master) make
/Applications/Xcode.app/Contents/Developer/usr/bin/make  all-recursive
Making all in src
  CC       ideviceinstaller-ideviceinstaller.o
ideviceinstaller.c:928:73: error: format specifies type 'long long' but the argument has type 'ssize_t' (aka 'long') [-Werror,-Wformat]
                                                                fprintf(stderr, "Error: wrote only %d of %" PRIi64 "\n", total, amount);
                                                                                                         ~~~                    ^~~~~~
ideviceinstaller.c:910:20: error: comparison of integers of different signs: 'off_t' (aka 'long long') and 'zip_uint64_t' (aka 'unsigned long long') [-Werror,-Wsign-compare]
                                        while (zfsize < zs.size) {
                                               ~~~~~~ ^ ~~~~~~~
2 errors generated.
make[2]: *** [ideviceinstaller-ideviceinstaller.o] Error 1
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
```

I tried to comment line 928 and change line 910 to
`while (zfsize < (unsigned) zs.size) {`,

but now I am getting this error when I trying to install IPA:

`ERROR: Install failed. Got error "ApplicationVerificationFailed" with code 0xe8008001: Failed to verify code signature of /private/var/installd/Library/Caches/com.apple.mobile.installd.staging/temp.DJ3jjb/extracted/Payload/PlexMobile.app : 0xe8008001 (An unknown error has occurred.)`

**Comment by nikias** [comment]
The error you're getting is from the device. You can't install IPAs with invalid/missing/expired code signature.

**Comment by nonpafos** [comment]
I'm having the same issue

**Comment by myqqai** [comment]
the same issue

**Comment by wangzi1325** [comment]
the same issue  
can't  compile

**Comment by Teketel** [comment]
I've made a quick fix for this issue.
clone my forked repo `git clone https://github.com/Teketel/ideviceinstaller.git`

**Comment by nikias** [comment]
Compilation issue fixed with #85. Rest is out of scope of this project (invalid signature).
